tools/cachemgr.cc

```diff
@@ -709,7 +709,7 @@ read_reply(int s, cachemgr_request * req)
             }
 
             istate = isBody;
-        /* yes, fall through, we do not want to loose the first line */
+        /* [[fallthrough]] fall through, we do not want to loose the first line */
```

Comment:
Same here, plus remove the duplicated "fall through"
```suggestion
        /* [[fallthrough]] we do not want to lose the first line */
```